build/shade/_k-standard-goals.shade

```diff
@@ -151,24 +150,11 @@ default SAMPLES_PROJECT_GLOB = "samples/*/*.csproj"
               }
           }
       });
-
-      projectGlobs.AddRange(srcProjects);
-    }
-    if (!BuildSrcOnly && Directory.Exists("test"))
-    {
-        projectGlobs.AddRange(Files.Include(TEST_PROJECT_GLOB).ToList());
-    }
-    if (!BuildSrcOnly && Directory.Exists("samples") && Directory.EnumerateFiles("samples", "*.csproj", SearchOption.AllDirectories).Any())
-    {
-        projectGlobs.AddRange(Files.Include(SAMPLES_PROJECT_GLOB).ToList());
     }
 
-    if (projectGlobs.Any())
+    foreach (var sln in Files.Include(Path.Combine(BASE_DIR, "*.sln")))
```

Comment:
I think this problem can be solved by the work Pranav is doing on https://github.com/aspnet/KoreBuild/pull/158. The default Korebuild build step could include:

```xml
<SolutionsToBuild Include="$(RepoRoot)\*.sln" />
```
MVC and other repos could customize by adding exclusions to the `build/repo.targets` file:

```xml
<SolutionsToBuild Remove="$(RepoRoot)\Mvc.NoFun.sln" />
```